src/inventory.py

```diff
@@ -1,14 +1,16 @@
-"""The Inventory singleton"""
+"""
+The Inventory singleton
+.. todo:: make imports dynamic, and watch out for frozen, like with messagetypes
+"""
 
-# TODO make this dynamic, and watch out for frozen, like with messagetypes
-import storage.sqlite
 import storage.filesystem
+import storage.sqlite
 from bmconfigparser import BMConfigParser
 from singleton import Singleton
 
 
 @Singleton
-class Inventory():
+class Inventory(object):
```

Comment:
Somehow this breaks `_command_inv` in `network.bmproto`. Please debug it or drop from this changeset.

Reply:
Maybe `@Singleton` conflicts with `object`.

Reply:
Yea looking at `src/singleton.py`, sounds plausible.